src/components/City/City.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './style.css';
+import cityData from './cityData';
+
+function City() {
+  return (
+    <div className="cities">
+      <div className="cities-card">
+        {Object.entries(cityData).map(([id, { name, latitude, longitude }]) => (
+          <li key={id} className="city">
+            <Link to={`/city/${name}`}>
+              <div className="city-details">
+                <h4>{name}</h4>
+                &nbsp;&nbsp;
+                <p>
+                  &#40;
+                  {latitude}
+                  ,&nbsp;
+                  {longitude}
+                  &#41;
+                </p>
+              </div>
+            </Link>
+          </li>
+        ))}
+      </div>
+    </div>
+  );
+}
```

Comment:
Great job creating this component. For the `City` component, kindly implement these two suggestions:
- [ ]  One of the major requirements for the `Home` page is that the list of categories should be filtered using the given parameters, e.g. category name. To implement this feature, kindly create a search bar that filters the list rendered when the state changes (input value is typed in the search bar). This [article](https://javascript.plainenglish.io/how-to-implement-a-search-bar-in-react-js-8cf8f5513b8e) will help you implement this feature. 
- [ ] Kindly make the design of your Home page to be identical to that of the design template provided. Kindly implement the grid layout used in the design, the unique background feature used for the grid cells and the icons that link to the `Detail` Page.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/169412242-ef4ef192-b191-4794-b970-e8b6c0d356f3.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/169412310-0f505118-3153-49f7-b786-7c827a64090f.png)